### ISSUE TITLE
Mask Site Forwarder

### ISSUE THREAD

**Issue by siddonj** [jlesage/docker-nginx-proxy-manager#35]
How do I mask a site that has been forwarded?

Ex. domainname.com forwards to organizr.domainname.com. How do I mask organizr.domainname.com

**Comment by jlesage** [comment]
What do you mean by "masking"?

**Comment by siddonj** [comment]
When I am forwarded to another subdomain  I  don’t want the subdomain to show in the address bar. I just want the actual domainname.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by jlesage** [comment]
Is the forwarding handled by Nginx Proxy Manager?

**Comment by siddonj** [comment]
Yes, forwarded to the Organizr application.  I have about 17 hosts that are reverse proxied.

________________________________

**Comment by jlesage** [comment]
So you are using a "redirection host"?  With this method I guess it's not possible to do what you want.

Why don't you setup a proxy host directly on domainname.com instead?

**Comment by siddonj** [comment]
I will give that a shot.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
